.devcontainer/docker-compose.yml

```diff
@@ -100,10 +105,13 @@ services:
                 aliases:
                     - keycloak.local
 
-    memcached:
-        container_name: memcached_${PROJ_PREFIX:-myproj}
-        image: memcached:latest
+    redis:
+        container_name: redis
```

Comment:
should probably keep the container name with the sufix
`redis_${PROJ_PREFIX:-myproj}`